src/applications/personalization/dashboard/components/notifications/Notifications.jsx

```diff
@@ -48,14 +48,13 @@ export const Notifications = ({
           </div>
         </DashboardWidgetWrapper>
       )}
-      {!notificationsError &&
-        debtNotifications.map((n, i) => (
-          <DebtNotification
-            key={i}
-            hasError={notificationsError}
-            notification={n}
-          />
-        ))}
+      {debtNotifications.map((n, i) => (
+        <DebtNotification
+          key={i}
+          hasError={notificationsError}

```

Comment:
So above you are checking for `notificationsError` and returning null in that case, but still passing `notificationsError` to `DebtNotifications` component here?

Does that need to be passed anymore to DebtNotifications?

Otherwise lgtm

Reply:
This wasn't part of it. It must have been transferred over. I'm going to replay my changes in a new branch